tests/lib/rules/valid-jsdoc.js

```diff
@@ -339,6 +339,35 @@ ruleTester.run("valid-jsdoc", rule, {
                     "astnode": "ASTNode"
                 }
             }]
+        },
+        {
+            code:
+            "/**\n" +
+            " * Test dash and slash.\n" +
+            " * @extends module:stb/emitter~Emitter\n" +
+            " */\n" +
+            "function foo() {}",
+            options: [{
+                "prefer": {
+                    "returns": "return"
```

Comment:
I know it doesn't hurt, but readers not in the know (such as me) might assume it's relevant to the test. This option imposes am unnecessary cognitive burden. Just my opinion but I think it should go.